### PR TITLE
Added E2E LLM test

### PR DESCRIPTION
**Pull request by AlexKoff88**

### CODE REVIEW

**Review by ilya-lavrenov — COMMENTED** [review]
I suppose the right place is GenAI repo? 
There we will switch to nightly releases soon, so can perform integration tests E2E

File: tests/llm/accruacy_conformance.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+]
+DEVICES = [
+    "CPU",
+    "GPU",
+]
+NUMBER_OF_SAMPLES = 15
+METRIC_OF_INTEREST = "similarity"
+
+REFERENCES = {
+    "llama": {"INT8": 0.93, "INT4": 0.93},
```
Comment (AlexKoff88) [inline tests/llm/accruacy_conformance.py:28]:
```suggestion
    "llama": {"INT8": 0.95, "INT4": 0.95},
```

File: tests/llm/accruacy_conformance.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    "qwen2": {"INT8": 0.70, "INT4": 0.70},
```
Comment (AlexKoff88) [inline tests/llm/accruacy_conformance.py:29]:
```suggestion
    "qwen2": {"INT8": 0.77, "INT4": 0.77},
```

**Comment by AlexKoff88** [comment]
Can we proceed with the merge? The test suite works on Linux CPU/GPU and WIndows CPU. There is a crash when using GPU on Windows but this is some runtime issue.

**Comment by AlexKoff88** [comment]
Details in Ref. 154161.

File: tests/llm/accruacy_conformance.py
[the remark this answers is not in the document]
Comment (AlexKoff88) [reply tests/llm/accruacy_conformance.py:1]: fixed

**Comment by AlexKoff88** [comment]
I think the PR is ready. Windows issue was fixed on the CI side.

**Comment by AlexKoff88** [comment]
Gentle ping to review and merge.

**Comment by AlexKoff88** [comment]
@ilya-lavrenov, @akladiev, need your approvals to merge.

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**